roles/post-db-restore/tasks/main.yml

```diff
@@ -99,6 +99,16 @@
   when: launch_tasks == ""
   notify: Clear site cache
 
+# chmod files directory so that there are not permissions issues when going from
+# AFS ACLs to Unix file permissions
+- name: Make all files readable on AFS
+  file:
+    path: "/afs/ir/dist/drupal/{{ site_prefix }}_{{ inventory_hostname }}/files/"
+    recurse: yes
+    mode: "u=rwX,g=rX,o=rX"
+  when:
+    afs_available == "TRUE" and chmod == "TRUE"
```

Comment:
Shouldn't we also do this on the local version?

Reply:
Or are we not running this on local anymore and should assume my-vbox or the people servers?

Reply:
Yeah, sure, wouldn't hurt on local too. Although I haven't run on local (my-stanford-vbox, in my case) other than for limited testing since we got people1/people2 set up. So much faster on those two boxes.

I got Bill set up on people1/2 and am steering him in that direction, so I'm kinda "meh" on this.